capsules/src/permissions.rs

```diff
@@ -0,0 +1,22 @@
+//! Simple capsule that checks if a permission is set to true or false
+
+#![forbid(unsafe_code)]
+
+pub struct Permissions {}
+
+impl Permissions {
+    pub fn new() -> Permissions {
+        Permissions {}
+    }
+
+    pub fn start(&self) {}
+
+    pub fn check(&self, permissions: u64, driver_num: usize) -> bool {
+        if let Some(bit) = crate::driver::get_permission_bit(driver_num) {
+            permissions & (1 << bit) == (1 << bit)
+        } else {
+            // driver not found
+            false
```

Comment:
This implies that drivers have to be whitelisted. I.e. if they aren't mentioned in the policy, they are default off. Not a bad policy, but should be noted somewhere (and maybe configurable?)